### ISSUE TITLE
median eminence placement

### ISSUE THREAD

**Issue by obophenotype-user** [obophenotype/mammalian-phenotype-ontology#1954]
Consider placing "abnormal median eminence morphology [MP:0009229]" as a direct child of "abnormal neurohypophysis morphology [MP:0004164]"(it is currently a child of "abnormal hypothalamus morphology [MP:0000837]").

see Adult Mouse Anatomy (http://www.informatics.jax.org/searches/AMA.cgi?id=MA:0000859) and UBERON:0002197 (http://purl.obolibrary.org/obo/UBERON_0002197)

UBERON applies "median eminence of neurohypophysis" as the primary term label. Exact synonyms include:

median eminence of tuber cinereum
median eminence of posterior lobe of pituitary gland
median eminence
eminentia mediana hypothalami
median eminence of hypothalamus

Thanks

Anna

Reported by: anna_anagnos

Original Ticket: [obo/mammalian-phenotype-requests/1954](https://sourceforge.net/p/obo/mammalian-phenotype-requests/1954)
